.vscode/launch.json

```diff
@@ -5,12 +5,7 @@
       "name": "Debug Web",
       "type": "node-terminal",
       "request": "launch",
-      "command": "cd apps/web && pnpm dev",
-      "serverReadyAction": {
-        "pattern": "- Local:.+(https?://.+)",
-        "uriFormat": "%s",
-        "action": "debugWithChrome"
-      }
```

Comment:
This tries to open chrome automatically when starting the server, which I don't want.
I am adding this to the repo because this is a standard can be used in any supported editor, not just mine.